lib/code_corps/policy/user_task.ex

```diff
@@ -15,10 +15,10 @@ defmodule CodeCorps.Policy.UserTask do
   alias Ecto.Changeset
 
   @spec create?(User.t, Changeset.t) :: boolean
-  def create?(%User{} = user, %Changeset{} = changeset) do
+  def create?(%User{} = user, params) do
     cond do
-      changeset |> get_task |> task_authored_by?(user) -> true
-      changeset |> get_task |> get_project |> contributed_by?(user) -> true
+      params |> get_task |> task_authored_by?(user) -> true
+      params |> get_task |> get_project |> contributed_by?(user) -> true
```

Comment:
I know this isn't quite relevant to your PR at hand, but I'm not sure now that I'm seeing it that I like how `contributed_by?/1` reads. Maybe we should consider an issue to rewrite to something like `has_contributor?/1`?